templates/zerver/app/index.html

```diff
@@ -91,7 +91,7 @@ <h3>{{ _('Loading...') }}</h3>
     </div>
     <div class="app-main">
         <div class="column-left">
-            {% include "zerver/app/left_sidebar.html" %}
+            <div id = "left-sidebar-holder"></div>
```

Comment:
I decided it was better to add `left-sidebar-container` as an ID on the `column-left` class here, to avoid creating an additional unnecessary div.

Reply:
Also note that we don't use space around the `=` in our HTML style.